Formula/ibex.rb

```diff
@@ -12,37 +22,59 @@ class Ibex < Formula
     sha256 "41327b6a0da9b8b2ed888c3c2c33d5f6ff061ea13eceb4b9e8b6e660049d624d" => :mountain_lion
   end
 
-  option "with-java", "Build Java bindings for Choco solver."
+  option "with-java", "Enable Java bindings for CHOCO solver."
+  option "with-ampl", "Use AMPL file loader plugin"
```

Comment:
Are there downsides to building this plugin by default? If it doesn't affect build time that much, doesn't take up much space, and doesn't create problems for users who don't need that feature, then let's enable it by default.